storage/redis/peer_store.go

```diff
@@ -8,48 +8,49 @@ import (
 
 	"github.com/chihaya/chihaya/bittorrent"
 	"github.com/chihaya/chihaya/storage"
-	"github.com/garyburd/redigo/redis"
+	redigo "github.com/garyburd/redigo/redis"
 )
 
-var namespacePrefix string
+var (
+	prefixedSeeder  string
```

Comment:
keep these as part of `peerStore` struct.

also I feel renaming them as `seederKeyPrefix` and `leecherKeyPrefix` is more readable.